stablehlo/tests/CheckOps.h

```diff
@@ -40,7 +42,8 @@ class CheckDialect : public Dialect {
 llvm::Error evalExpectAlmostEqConstOp(const Tensor &lhs, ElementsAttr value);
 llvm::Error evalExpectAlmostEqOp(const Tensor &lhs, const Tensor &rhs);
 llvm::Error evalExpectEqConstOp(const Tensor &lhs, ElementsAttr value);
-llvm::Error evalExpectEqOp(const Tensor &lhs, const Tensor &rhs);
+llvm::Error evalExpectEqOp(const InterpreterValue &lhs,
+                           const InterpreterValue &rhs);
```

Comment:
I don't think we need to change the signature of this function, as well as add extra includes?